lib/make-error.js

```diff
@@ -1,3 +1,5 @@
+var MulterError = require('./multer-error')
+
 var errorMessages = {
```

Comment:
Could we move this map to the `multer-error.js` file, remove `message` as a constructor argument to `MutlerError`, replace all calls to `makeError` with `new MulterError`, and finally remove this file? Thanks ☺️

Reply:
Great idea! Just did it. Please, take a look.